jest.config.json

```diff
@@ -9,5 +9,9 @@
     "coverageDirectory": "<rootDir>/../coverage",
     "setupFilesAfterEnv": ["<rootDir>/../jest.setup.ts"],
     "testMatch": ["**/__tests__/**/*.test.(js|ts)?(x)"],
-    "testPathIgnorePatterns": ["ssr.test.tsx"]
+    "testPathIgnorePatterns": ["ssr.test.tsx"],
+    "watchPlugins": [
```

Comment:
added bonus of this PR - it's a super useful jest plugin, which once was in the core, no idea why they have removed it, see a gif-demo here: https://github.com/jest-community/jest-watch-typeahead

If you want I can remove this or move to another PR